editor/export/project_export.cpp

```diff
@@ -102,11 +102,18 @@ void ProjectExportDialog::_notification(int p_what) {
 		case NOTIFICATION_THEME_CHANGED: {
 			duplicate_preset->set_icon(presets->get_editor_theme_icon(SNAME("Duplicate")));
 			delete_preset->set_icon(presets->get_editor_theme_icon(SNAME("Remove")));
+			patch_add_btn->set_icon(get_editor_theme_icon(SNAME("Add")));
+
+			Control *panel = custom_feature_display->get_parent_control();
+			if (panel) {
+				panel->add_theme_style_override(SceneStringName(panel), get_theme_stylebox("bg", "Tree"));
+			}
```

Comment:
What's this?

Reply:
I'm not sure to be honest. It looks unrelated to this PR, so I went ahead and removed it. Maybe @PoqXert can shed some light on why this was in there.